runtime/compiler/runtime/DataCache.cpp

```diff
@@ -293,6 +293,19 @@ TR_DataCache* TR_DataCacheManager::allocateNewDataCache(uint32_t minimumSize)
                dataCache->_status = 0;
                dataCache->_vmThread = NULL;
                dataCache->_allocationMark = dataCacheSeg->heapAlloc;
+
+#ifdef LINUX
+               if (OMR::RSSReport::instance())
+                  {
+                  J9JavaVM * javaVM = jitConfig->javaVM;
+                  PORT_ACCESS_FROM_JAVAVM(javaVM); // for j9vmem_supported_page_sizes
+
+                  dataCache->_rssRegion = new (PERSISTENT_NEW) OMR::RSSRegion("data cache", dataCacheSeg->heapBase, segSize, OMR::RSSRegion::lowToHigh,
```

Comment:
dataCache->_rssRegion needs to be set to NULL, if it's not used.

Reply:
What happens if the allocation fails?

Reply:
Do you mean if allocation of` OMR::RSSRegion` fails?

Reply:
Added default constructor and initiliazed `_rssRegion`. Thanks for noticing!